data/faq/steal_project_idea.md

```diff
@@ -0,0 +1,9 @@
+The whole point of "Open Source Software" is to encourage celebration of
+complete independence in sharing and writing code.If you do honor original
+authors, licenses, and the "creator" of an original idea, there is no shame in
+saying "he had a great idea, I think I can implement it". It's called
+"collaboration". On the plus side, we will always take into account who came
+up with something and if you help others succeed, we'll definitely credit you
+for that. Most importantly, we want our contributors to work as a team and not
+like some "lone hackers" keeping secrets from fellow contributors, and sending
+unnecessary private messages to the project mentors.
```

Comment:
Please point out that these is such a thing as stealing in OS, it is when you don't credit resources, authors, or disregard the license.